src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml

```diff
@@ -93,8 +93,15 @@ limitations under the License.
     <attribute side="server" code="0x06" define="ACTIVE_RADIO_FAULTS" type="ARRAY" entryType="RadioFaultEnum" writable="false" optional="true">ActiveRadioFaults</attribute>
     <attribute side="server" code="0x07" define="ACTIVE_NETWORK_FAULTS" type="ARRAY" entryType="NetworkFaultEnum" writable="false" optional="true">ActiveNetworkFaults</attribute>
     <attribute side="server" code="0x08" define="TEST_EVENT_TRIGGERS_ENABLED" type="boolean" writable="false" optional="false">TestEventTriggersEnabled</attribute>
-    <!-- TODO(#30002): Remove AverageWearCount -->
-    <attribute side="server" code="0x09" define="AVERAGE_WEAR_COUNT" type="int32u" writable="false" default="0x00000000" optional="true">AverageWearCount</attribute>
+    <!--
+        WARNING !!!!! Attribute 0x0009 (previously AverageWearCount, see #30002/#29285)
```

Comment:
@robszewczyk Waiting for your approval to merge. Resolve comment when satisfied.

Reply:
approved. resolving this comment.  The other seems to be for @bzbarsky-apple to resolve